src/MediaAccessibility/MediaAccessibility.cs

```diff
@@ -262,6 +262,34 @@ public static MACaptionAppearanceTextEdgeStyle GetTextEdgeStyle (MACaptionAppear
 			behavior = (MACaptionAppearanceBehavior) (int) b;
 			return (MACaptionAppearanceTextEdgeStyle) (int) rv;
 		}
+
+		[TV (13,0), Mac (10,15), iOS (13,0)]
+		[DllImport (Constants.MediaAccessibilityLibrary)]
+		static extern void MACaptionAppearanceDidDisplayCaptions (IntPtr /* CFArratRef */ strings);
+
+		[TV (13,0), Mac (10,15), iOS (13,0)]
+		public static void DidDisplayCaptions (string[] strings)
+		{
+			if ((strings == null) || (strings.Length == 0))
+				MACaptionAppearanceDidDisplayCaptions (IntPtr.Zero);
+			else {
+				using (var array = NSArray.FromStrings (strings))
+					MACaptionAppearanceDidDisplayCaptions (array.Handle);
+			}
+		}
+
+		[TV (13,0), Mac (10,15), iOS (13,0)]
+		public static void DidDisplayCaptions (NSAttributedString[] strings)
+		{
+			// CFAttributedString is “toll-free bridged” with its Foundation counterpart, NSAttributedString.
+			// https://developer.apple.com/documentation/corefoundation/cfattributedstring?language=objc
+			if ((strings == null) || (strings.Length == 0))
+				MACaptionAppearanceDidDisplayCaptions (IntPtr.Zero);
```

Comment:
Why do we use `IntPtr.Zero` in place of `null`?

Reply:
It's basically all the same, just like `0` would be - but
* C# is type safe (unlike C/C++); and
* `IntPtr` is a `struct` not a reference type so it can't be set to `null`; and

We could use pointers in C# but that would expose a lot of `unsafe` code everywhere... which does not make great API.

We could have a 2nd `[DllImport]` with a different signature - but that would bloat the code.

Reply:
Thank you for the detailed explanation, makes a lot of sense! ❤️